assets/js/application.js

```diff
@@ -10,8 +10,8 @@
 // Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
 // about supported directives.
 //
-//= require jquery-1.7.1.min
-//= require jquery-ui-1.8.18.custom.min
+//= require jquery-3.7.1.min
+//= require jquery-ui-1.14.1.min
```

Comment:
It does not make much sense to split the JQuery version updates in `application.js` from the ones in the header.

I am glad that you dropped the unrelated change from this commit where `jquery.defaultvalue` was dropped. But it seems you no longer drop this altogether? Is it still supported? Does it support the newest JQuery version?